stages/org.osbuild.zipl-secex.inst

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python3
+"""
+Install the Z Initial Program Loader (zipl)
+
+This will install the Z Initial Program Loader for the
+specified `kernel` with "IBM Secure Execution" enabled.
+The other required parameter are
+- `location`, which specifies is the offset of the partition containing `/se`
+within the device that is specified via `disk`.
+- `ignition_gpg_pubkey`, which specifies filename for public GPG key
+- `hostkey`, which specifies Hostkey for given IBM Z machine
+- `build_id`, which specifies CoreOS build
```

Comment:
Honestly for me the architecture of SECEX is still fairly opaque. If we got a good architecture diagram/description here of the way the disk is expected to be setup it might help casual readers understand a lot more here. OR if there is some documentation we can point to that fills in those gaps, that could suffice.